### PR TITLE
move search.md

### PR DESCRIPTION
**Pull request by kbhawkey**
Investigating whether the site should use the same placement of the search page as the Docsy theme.
The Docsy theme sample site lists `search.md` directly under `content/en`.
With this change, the search page no longer displays as a `docs` page.
This also helps to clean up single files from docs/.

### CODE REVIEW

**Comment by zacharysarah** [comment]
/lgtm

Pinging @celestehorgan for approval

/assign @celestehorgan

**Comment by tengqm** [comment]
@kbhawkey Please remove the `[wip]` if you are okay for this to be merged.

**Comment by kbhawkey** [comment]
Let me look at this again for a few minutes. My main reason for starting this change was to remove the search file
from the docs root (and prevent the file from displaying in the local build) and investigate whether the search bar
could be moved around.

File: assets/js/search.js [hunk 1]
```diff
@@ -23,8 +23,7 @@ limitations under the License.
           }
 
           var query = $(this).val();
-          var searchPage = "{{ "docs/search/" | absURL }}?q=" + query;
-          document.location = searchPage;
+          document.location = "{{ "search/" | absURL }}?q=" + query;
```
Comment (kbhawkey) [inline assets/js/search.js:26]: At the moment, the source is the same as found in the theme. The file could be removed.

File: layouts/_default/search.html [hunk 2]
```diff
@@ -1,6 +1,11 @@
 {{ define "main" }}
 <section class="row td-search-result">
 <div class="col-12 col-md-8 offset-md-2">
+	<form class="td-sidebar__search d-flex align-items-center">
+		{{ partial "search-input.html" . }}
+		<button class="btn btn-link td-sidebar__toggle d-md-none p-0 ml-3 fas fa-bars" type="button" data-toggle="collapse" data-target="#td-section-nav" aria-controls="td-docs-nav" aria-expanded="false" aria-label="Toggle section navigation">
```
Comment (kbhawkey) [inline layouts/_default/search.html:6]: Testing the placement of a search bar on the search page.

File: static/js/search.js [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
         for(var i = 1; i <= 10; i++){
             if(i > pages) break;
-            pageAnchors += '<a class="bing-page-anchor" href="/docs/search/?q='+searchTerm+'&page='+i+'">';
+            pageAnchors += '<a class="bing-page-anchor" href="search/?q='+searchTerm+'&page='+i+'">';
```
Comment (sftim) [inline static/js/search.js:20]: How come this isn't:
```suggestion
            pageAnchors += '<a class="bing-page-anchor" href="/search/?q='+searchTerm+'&page='+i+'">';
```

**Review by sftim — COMMENTED** [review]
I've got one query, I'm wondering if the Bing code should link to `search?q=…` or to `/search?q=…`

I think the former could lead to URLs like: https://kubernetes.io/search/search?q=…

File: static/js/search.js (hunk 3 above)
Comment (kbhawkey) [reply static/js/search.js:20]: Yes, I will correct this. Thanks.
It was hard to find the numbers, at the bottom of the Bing search results page, which link to the next page results.

File: layouts/_default/search.html (hunk 2 above)
Comment (kbhawkey) [reply layouts/_default/search.html]: Probably need to adjust the formatting on this page.

**Comment by sftim** [comment]
Just so you know, GitHub is struggling (at the moment) to render this PR's changes.
I plan to check again when things are healthier there.

**Comment by kbhawkey** [comment]
https://deploy-preview-23083--kubernetes-io-master-staging.netlify.app/search/?q=kubectl

**Comment by zacharysarah** [comment]
Tangentially: After this merges, I'd like to investigate whether it's possible to render a prettier results page. That may well be an upstream issue.

Leaving approve to another chair/tech lead, but otherwise:

/lgtm

**Comment by sftim** [comment]
/approve

Thanks for the improvement!

**Pull request merged**